services/src/main/webapp/WEB-INF/config.properties

```diff
@@ -6,3 +6,5 @@ usersavedselection.watchlist.searchurl=catalog.search#/search?_uuid={{filter}}
 
 # Define the link to each record sent by email by the watchlist notifier
 usersavedselection.watchlist.recordurl=api/records/{{index:_uuid}}
+
+bot.regexpfilter=${bot.regexpfilter}
```

Comment:
Looks like this should be regexp**F**ilter to be consistent.
Also I don't believe it is recommended that the property name be equal to the placeholder.  This may be what is causing the Circular placeholder reference